README.md

```diff
@@ -1,17 +1,277 @@
+- [MinimumSecondaryActionsCompleted](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L33) : Make sure that it prompts for review only if the number of completed secondary actions meets the minimum.
+- [MinimumApplicationLaunchCount](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L49) : Make sure that it prompts for review only if the number of times the application has been launched meets the required minimum.
+- [MinimumTimeElapsedSinceApplicationFirstLaunch](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L65) : Make sure that it prompts for review only if the time elapsed since the first application launch meets the required minimum.
+- [Custom](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L83) : Custom condition made with a synchronous lambda function.
+- [CustomAsync](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L97) : Custom asynchronous condition made with an asynchronous lambda function.
+
+### Add Custom Conditions
+
+To create custom review conditions, you have to use `ReviewConditionsBuilder.Custom` and `ReviewConditionsBuilder.CustomAsync` and provide them with a function directly instead of a condition. Also you can create extensions for `IReviewConditionsBuilder` and add a new condition to the builder. To create a review condition, you can use both `SynchronousReviewCondition` and `AsynchronousReviewCondition` you need to provide them with a function.
+
+``` cs
+namespace ReviewService;
+
+/// <summary>
+/// Extensions for <see cref="IReviewConditionsBuilder{TReviewSettings}"/>.
+/// </summary>
+public static partial class ReviewConditionsBuilderExtensions
+{
+    /// <summary>
+    /// The application onboarding must be completed.
+    /// </summary>
+    /// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+    /// <param name="builder">The builder.</param>
+    /// <returns><see cref="IReviewConditionsBuilder{TReviewSettings}"/>.</returns>
+    public static IReviewConditionsBuilder<TReviewSettings> ApplicationOnboardingCompleted<TReviewSettings>(this IReviewConditionsBuilder<TReviewSettings> builder)
+        where TReviewSettings : ReviewSettingsCustom
+    {
+        builder.Conditions.Add(new SynchronousReviewCondition<TReviewSettings>(
+            (reviewSettings, currentDateTime) => reviewSettings.HasCompletedOnboarding is true)
```

Comment:
Bit nitpicking here, but I don't think we need to explicitly say `reviewSettings.HasCompletedOnboarding is true`, just `reviewSettings.HasCompletedOnboarding` is good enough in my opinion

Reply:
Don't you think it's more readable that way? I mean it does make sense but I feel like the documentation code should be as clear as possible?

Reply:
Yeah I guess that's fair in this contexte